src/assets/wise5/components/common/feedbackRule/feedback-rule-help/feedback-rule-help.component.html

```diff
@@ -46,6 +46,13 @@ <h3 i18n>Terms</h3>
         Ex: <b>ideaCountEquals(2)</b> evaluates to true if the student had exactly 2 ideas.
       </td>
     </tr>
+    <tr>
+      <td class="term">isSubmitNumber(X)</td>
+      <td i18n>
+        Evaluates to true if this is the student’s X-th attempt.<br />
```

Comment:
I noticed that a lot of the apostrophes in this file are fancy apostrophes. Should we change all of them to plain apostrophes?
Fancy `’`
Plain `'`

Reply:
Good catch. Must have been from when I did copy-paste from Word.